packages/wonder-blocks-button/generated-snapshot.test.js

```diff
@@ -1,4 +1,3 @@
-// @flow
```

Comment:
🛑 Why did these go away?

Reply:
I can't create an example in `docs.md` that includes `extends React.Component<Props, State>` b/c `styleguidist` can't handle that.  The generated snapshot then contains code that `flow` won't pass.